app/src/main/java/com/example/habitsmasher/UserRegistrationFragment.java

```diff
@@ -193,6 +210,39 @@ public void onComplete(@NonNull Task<Void> task) {
                                    }
                              }
                          });
+        addUserImageToDatabase(user.getId());
+    }
+
+    /**
+     * @param userId The ID of the user for which the picture is uploaded
+     */
+    public void addUserImageToDatabase(String userId) {
+        Uri toUpload = _selectedImage;
+
+        if (toUpload == null) {
+            toUpload = Uri.parse("android.resource://com.example.habitsmasher/drawable/placeholder_profile_picture");
```

Comment:
Can we put this hardcoded string path somewhere else? String resources, a constant, anything level of abstraction will do!